requirements-base.txt

```diff
@@ -245,7 +245,7 @@ preshed==3.0.8
     # via
     #   spacy
     #   thinc
-protobuf==4.21.12
+protobuf==4.22.0
```

Comment:
@mvilanova recently downgraded `protobuf` because it caused an issue with one of the plugins. Flagging this one.